src/Glob.php

```diff
@@ -61,7 +61,7 @@ protected static function systemGlob($pattern, $flags)
                 self::GLOB_NOSORT   => GLOB_NOSORT,
                 self::GLOB_NOCHECK  => GLOB_NOCHECK,
                 self::GLOB_NOESCAPE => GLOB_NOESCAPE,
-                self::GLOB_BRACE    => GLOB_BRACE,
+                self::GLOB_BRACE    => defined('GLOB_BRACE') ? GLOB_BRACE : 0,
```

Comment:
(*whisper* same thing about that link you just posted *whisper*).

Thanks btw :-)